1875_Mp3_Player_Clone.js

```diff
@@ -0,0 +1,316 @@
+console.log(`                                                                 USABLE FUNCTIONS: 
```

Comment:
why?

Reply:
lol .. just so it be more comfortable for the programmer to have all the usable fucntions n the prgram in front of his eyes.(when code has no errors)... i knw its making file heavier... but idk i just wanted it for fun